cmd/translate/main_grpc_integration_test.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	pb "go.expect.digital/translate/pkg/server/translate/v1"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/status"
+)
+
+func TestMain(m *testing.M) {
+	go main()
+
+	// Needed to prevent panic, when service is not ready to receive requests
+	time.Sleep(time.Millisecond * 50)
+
+	os.Exit(m.Run())
+}
+
+func createConnection(ctx context.Context, t *testing.T) *grpc.ClientConn {
+	t.Helper()
+
+	conn, err := grpc.DialContext(
+		ctx,
+		"localhost:8080",
+		grpc.WithTransportCredentials(insecure.NewCredentials()),
```

Comment:
Wouldn't it be better to find a free port? Port 8080 might not always be available.

Reply:
I believe this is not a priority at the momennt